src/test/java/org/broadinstitute/hellbender/tools/walkers/variantutils/LeftAlignAndTrimVariantsIntegrationTest.java

```diff
@@ -0,0 +1,62 @@
+package org.broadinstitute.hellbender.tools.walkers.variantutils;
+
+import org.broadinstitute.barclay.argparser.CommandLineException;
+import org.broadinstitute.hellbender.CommandLineProgramTest;
+import org.broadinstitute.hellbender.cmdline.StandardArgumentDefinitions;
+import org.broadinstitute.hellbender.testutils.IntegrationTestSpec;
+import org.testng.annotations.DataProvider;
+import org.testng.annotations.Test;
+
+import java.io.IOException;
+import java.nio.file.*;
+import java.util.*;
+
+
+public class LeftAlignAndTrimVariantsIntegrationTest extends CommandLineProgramTest {
```

Comment:
I might consider trying to find/construct some larger test data than these alleles. The depth of the tests in GATK3 for this tool appears to have been pretty light and we could use perhaps a much larger snippet of VCF or some more pathological sites to demonstrate that the code is behaving properly. I'm not sure who to ask for that though.

Reply:
Ya I agree.  I will hunt for some pathologically long repeats/strs in the reference and add some tests using them.